tsdb/engine/tsm1/compact.go

```diff
@@ -1752,7 +1772,8 @@ RETRY:
 		}
 
 		if iter.Err() != nil {
-			k.err = iter.Err()
+			k.err = errBlockRead{k.currentTsm, iter.Err()}
+			panic("block read")
```

Comment:
If we panic here do we end up processing the error at all? If we could skip the `panic` that would be awesome and just bubble up the err here, right?

Reply:
oops - this was accidentally left in.